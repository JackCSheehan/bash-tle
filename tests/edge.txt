GPS BIIR-2  (PRN 13)    
1 24876U 97035A   22260.10125264 -.00000029  00000-0  00000-0 0  9995
2 24876  55.5199 152.0985 0062576  52.2989 308.3296  2.00563211184503