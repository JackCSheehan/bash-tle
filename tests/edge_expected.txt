GPS BIIR-2 (PRN 13),24876,U,97,035,A,22,260.10125264,-.00000029,00000-0,00000-0,0,999,5,24876,55.5199,152.0985,0062576,52.2989,308.3296,2.00563211,18450,3
